pypowsybl/network.py

```diff
@@ -4751,3 +4751,38 @@ def _get_c_dataframes_and_add_voltage_level_id(network: Network, dfs: _List[_Opt
         else:
             c_dfs.append(_create_c_dataframe(df, metadata[i]))
     return c_dfs
+
+def replace_tee_point_by_voltage_level_on_line(network: Network, line1z_id: str, linez2_Id: str, linezp_id: str,
+                                  voltage_level_id: str, bbs_or_bus_id: str, line1c_id: str, linec2_id: str,
+                                  line1c_name: str = None, linec2_name: str = None) -> None:
+    """
+    This method transform the action done in the create_line_on_line function into the action done in the connect_voltage_level_on_line class :
+    it replaces 3 existing lines (with the same voltage level at one of their side (tee point)) with two new lines,
+    and removes the tee point
+    Args:
+        line1z_id : The ID of the existing line connecting the first voltage level to the tee point
+        linez2_Id : The ID of the existing line connecting the tee point to the second voltage level
+        linezp_id : The ID of the existing line connecting the tee point to the attached voltage level
+        voltage_level_id : The ID of the existing attached voltage level
+        bbs_or_bus_id : The ID of the existing bus or bus bar section in the attached voltage level voltageLevelId,
+          where we want to connect the new lines line1C and lineC2
+        line1c_id : The ID of the new line connecting the first voltage level to the attached voltage level
+        linec2_id : The ID of the new line connecting the second voltage level to the attached voltage level
+        line1c_name : The optional name of the new line connecting the first voltage level to the attached voltage level
+        linec2_name : The optional name of the new line connecting the second voltage level to the attached voltage level
+
+        VL1 ---------- tee point ---------- VL2                            VL1 ---------- attached voltage level ---------- VL2
```

Comment:
We need to check how this renders in the HTML doc, I have a bad feeling :)

Reply:
I understand the bad feeling, I'll remove it as suggested by Anne we will put that in the readthedoc.